acquia/acquia_cms
# 1070 Adds Nextjs to ACMS Headless module

-- Installs nextjs module(s) as dependency
-- Adds permissions
-- Apply patches via module composer.json
-- Pins Nextjs to version 1 in main composer.json
-- Updates composer.lock file
-- DOES NOT remove dependencies (nextjs) on uninstall

**Motivation**

Fixes #1070

**Proposed changes**


**Alternatives considered**

None

**Testing steps**

Install and uninstall ACMS Headless w/o errors
Check permissions
Add a nextjs headless site to see if patches applied

**Merge requirements**
- [ ] _Enhancement_
- [ ] Manual testing by a reviewer (see instructions above)
